main.js

```diff
@@ -1 +1,109 @@
+      buttons1: 'link',
+      button2: 'link2',
+    },
+    {
+      title: 'Keeping track of hundreds of components - 5',
+      image: 'img/test.png',
+      badge1: 'Ruby on rails',
+      badge2: 'CSS',
+      badge3: 'JavaScript',
+      badge4: 'HTML',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+      buttons1: 'link',
+      button2: 'link2',
+    },
+    {
+      title: 'Keeping track of hundreds of components - 6',
+      image: 'img/test.png',
+      badge1: 'Ruby on rails',
+      badge2: 'CSS',
+      badge3: 'JavaScript',
+      badge4: 'HTML',
+      description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+      button1: 'link',
+      button2: 'link2',
+    },
+  ];
+  function toggleBarWorks(workNumber) {
+    const y = document.getElementById('mobile-popup-section');
+    const i = workNumber;
+    document.getElementById('img-pop').src = projects[i].image;
+    document.getElementById('h3-popup').innerHTML = projects[i].title;
+    document.getElementById('badge1').innerHTML = projects[i].badge1;
+    document.getElementById('badge2').innerHTML = projects[i].badge2;
+    document.getElementById('badge3').innerHTML = projects[i].badge3;
+    document.getElementById('badge4').innerHTML = projects[i].badge4;
+    document.getElementById('p-pop').innerHTML = projects[i].description;
+    document.getElementById('btn-1').href = projects[i].button1;
+    document.getElementById('btn-2').href = projects[i].button2;
+
+    if (y.style.display === 'flex') {
+      y.style.display = 'none';
+    } else {
+      y.style.display = 'flex';
+    }
+  }
+
+  document.getElementById('xbut').addEventListener('click', () => { toggleBarWorks(0); });
+  document.getElementById('btnPopup1').addEventListener('click', () => { toggleBarWorks(0); });
+  document.getElementById('btnPopup2').addEventListener('click', () => { toggleBarWorks(1); });
```

Comment:
- [ ] Kindly fix the alignmenet the technologies to be in the beginning so I can see them in the middle so please fix that I will show an image to point what I am talking about!

- [ ] Kindly add some textdecoration for the buttons since it is not found in the original design 💯😊
- Your design
![image](https://user-images.githubusercontent.com/58553711/188770290-e65627a3-444e-4125-9a2b-7e2a59c9e4dc.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/188770345-4c6951e2-801c-47c2-8e22-d10614ca4baa.png)